catalogue/webapp/pages/work.js

```diff
@@ -186,13 +247,147 @@ export const WorkPage = ({ work }: Props) => {
           </div>
         </div>
       </Space>
-      <WorkDetails
-        work={work}
-        itemUrl={itemUrlObject}
-        iiifPresentationManifest={iiifPresentationManifest}
-        childManifestsCount={childManifestsCount}
-        imageCount={imageTotal}
-      />
+      <TogglesContext.Consumer>
+        {({ availableOnline }) => (
+          <>
+            {!availableOnline && (
+              <>
+                {!imageUrl && (
+                  <ManifestContext.Provider
+                    value={firstChildManifest || iiifPresentationManifest}
```

Comment:
Is the logic here that we want the first child manifest if it exists.

But we need to get the original one before we know that?

If so, could we not write a method that does this? We could have it all contained in a specialised ManifestContext component.